components/Markdown/Markdown.css

```diff
@@ -1,55 +1,259 @@
-.msrd .msrd-Markdown h1 {
-  font: var(--font-12);
-  font-weight: 800;
-  letter-spacing: var(--font-12-letter-spacing);
-  margin: 0;
-  margin-bottom: var(--space-vertical-04);
+.msrd .msrd-Markdown {
+  font: var(--font-03);
+  letter-spacing: var(--font-03-letter-spacing);
 }
 
-.msrd .msrd-Markdown h2 {
-  font: var(--font-10);
-  font-weight: 800;
-  letter-spacing: var(--font-10-letter-spacing);
-  margin: 0;
-  margin-bottom: var(--space-vertical-04);
+.msrd .msrd-Markdown--measured {
+  max-width: 34em;
 }
 
-.msrd .msrd-Markdown h3 {
-  font: var(--font-08);
-  font-weight: 800;
-  letter-spacing: var(--font-08-letter-spacing);
-  margin: 0;
-  margin-bottom: var(--space-vertical-04);
+.msrd .msrd-Markdown--inline {
+  font: inherit;
+  letter-spacing: inherit;
+  max-width: 100%;
 }
 
-.msrd .msrd-Markdown h4 {
-  font: var(--font-06);
+@media (min-width: 48em) {
+  .msrd .msrd-Markdown {
+    font: var(--font-04);
+    letter-spacing: var(--font-04-letter-spacing);
+  }
+
+  .msrd .msrd-Markdown--inline {
+    font: inherit;
+    letter-spacing: inherit;
+  }
+}
+
+.msrd .msrd-Markdown blockquote {
+  margin-bottom: var(--space-vertical-03);
+  margin-left: 1em;
+  margin-right: 0;
+  margin-top: 0;
+}
+
+.msrd .msrd-Markdown blockquote p {
+  margin-bottom: var(--space-vertical-03);
+  quotes: "“" "”" "‘" "’";
+}
+
+.msrd .msrd-Markdown blockquote p:first-child::before {
+  content: open-quote;
+  margin-left: -0.66ch;
+}
+
+.msrd .msrd-Markdown blockquote p:first-child::after {
+  content: close-quote;
+}
+
+.msrd .msrd-Markdown blockquote p + p:last-child {
+  text-align: right;
+  font: var(--font-02);
+}
+
+@media (min-width: 48em) {
+  .msrd .msrd-Markdown blockquote p + p:last-child {
+    font: var(--font-03);
+  }
+}
+
+.msrd .msrd-Markdown code {
+  background: var(--color-grey-9);
+  border-radius: 0.25em;
+  font-size: var(--font-02-size);
+  font-family: ui-monospace, 'Cascadia Code', 'Source Code Pro', Menlo, Consolas, 'DejaVu Sans Mono', monospace;
+  font-weight: inherit;
+  letter-spacing: 0;
+  line-height: var(--font-02-line-height);
```

Comment:
Would it work to use the `font` property and override `font-family` here, rather than explicitly setting `line-height`?

Reply:
You're right that would make more sense 👍🏻